libs/langchain/langchain/utilities/pubmed.py

```diff
@@ -158,15 +158,30 @@ def retrieve_article(self, uid: str, webenv: str) -> dict:
         return self._parse_article(uid, text_dict)
 
     def _parse_article(self, uid: str, text_dict: dict) -> dict:
-        ar = text_dict["PubmedArticleSet"]["PubmedArticle"]["MedlineCitation"][
-            "Article"
-        ]
-        summary = "\n".join(
-            [
-                f"{txt['@Label']}: {txt['#text']}"
-                for txt in ar.get("Abstract", {}).get("AbstractText", [])
-                if "#text" in txt and "@Label" in txt
+        try:
```

Comment:
Could you add a test here: 

https://github.com/langchain-ai/langchain/blob/eugene%2Fautomaton_variant_3/libs/langchain/tests/integration_tests/utilities/test_pubmed.py#L13-L13


Ideally, we would have a test fixture for the actual parsing logic

cc @leo-gan